pocs/tests/test_pocs.py

```diff
@@ -311,6 +323,43 @@ def test_run_complete(pocs):
 
     pocs.run(exit_when_done=True, run_once=True)
     assert pocs.state == 'sleeping'
+    pocs.power_down()
+
+
+def test_run_interrupt_with_reschedule_of_target(observatory):
```

Comment:
Probably very worthwhile, but is this related?

Reply:
Hm, not sure why this is in this PR. 😕